README.md

```diff
@@ -1,2 +1,11 @@
 # hw01
-Repository for homework 1
```

Comment:
Evaluation HW-01 for Sophia Kierstead
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   NA        |              |            |
| **Coding strategy**         |     NA      |              |            |
| **Presentation**            |    I think it's good in that it uses the required syntax well. I will suggest that some of hashtags are no that needed. The text itself may not need to be a new heading, and it seems to be not necessary to make "A few things I'm involved in" and "some random facts" different levels of heading. (Sometimes heading will make thing a little bit confusing, as I really not sure the list syntax work well or not, so I did check the space) But overall good bio.  |              |            |
| **Achievement, creativity** |    All is good besides the heading issue.  And it may be better to make the cat bio a separate section! It is funny.   |              |            |
| **Ease of access**          |    NA       |              |            |